src/api/Login.js

```diff
@@ -0,0 +1,35 @@
+import {Axios} from  './Axios';
+
+
+export const AxiosLogin = (data, loginToggle, goMain ) => {
+
+    const LoginSuccess = (accessToken) => {
+        window.localStorage.setItem("accessToken", JSON.stringify(accessToken))
+        const LocalAccessToken = JSON.parse(localStorage.getItem('accessToken'));
+
+        if (LocalAccessToken)
+        {
+            goMain()
+            loginToggle();
+        }
+    }
+
+    Axios.post(`/api/auth/login`, {
+        id: data.id,
+        password: data.password,
```

Comment:
위에서 {id,password} = data와 같이 구조분해할당을 했더라면, 만약 나중에 보내야하는 데이터의 형태가 id,password 2개가 아닌 email, validate, grade ... 와 같이 계속해서 속성수가 늘어난다고 해도 유지보수가 편하겠죠?

뿐만아니라 구조분해할당이 이루어졌다면 해당 객체에서 Key와 Value의 값이 동일하므로 
```suggestion
    Axios.post(`/api/auth/login`, {
       id,
       password
```
와 같이 줄일 수 있습니다.
더 깔끔하죠?